src/librustc_passes/stability.rs

```diff
@@ -438,7 +438,7 @@ fn new_index(tcx: TyCtxt<'tcx>) -> Index<'tcx> {
         // If the `-Z force-unstable-if-unmarked` flag is passed then we provide
         // a parent stability annotation which indicates that this is private
         // with the `rustc_private` feature. This is intended for use when
-        // compiling librustc crates themselves so we can leverage crates.io
+        // compiling librustc_middle crates themselves so we can leverage crates.io
```

Comment:
NB: this said "librustc crates". It could now be "`librustc_*` crates" or just "rustc crates".